lib/net/pool.js

```diff
@@ -2012,6 +2123,99 @@ class Pool extends EventEmitter {
     peer.sendCFCheckpt(packet.filterType, packet.stopHash, filterHeaders);
   }
 
+  /**
+   * Handle peer `CFCheckpt` packet.
+   * @method
+   * @private
+   * @param {Peer} peer
+   * @param {CFCheckptPacket} packet
+   */
+
+  async handleCFCheckpt(peer, packet) {
+    if (!this.options.neutrino) {
```

Comment:
Should resemble spv logic like below, although I think banning is too extreme personally, but we can discuss.

```js
    if (!this.options.spv) {
      this.logger.warning(
        'Peer sent unsolicited merkleblock (%s).',
        peer.hostname());
      peer.increaseBan(100);
      return;
    }
```